src/main/resources/i18n/messages.properties

```diff
@@ -351,24 +351,11 @@ project.members.add.success={0} was added to project {1}.
 project.members.add.error=An error occurred while adding the user to the project.
 project.members.add.required=User and project role must be selected before submitting.
 project.members.remove.modal.confirm=You are about to remove <strong>{0}</strong> from the project.
-
 # Project Metadata
 project.metadata.id=Project Id
 project.metadata.createdDate=Created Date
 project.metadata.modifiedDate=Modified Date
 
-# Users Page
```

Comment:
Since `users.username` and `users.role` were removed from the file the translations aren't found on the user create and user edit pages. So you will need to update those pages to use the new translation keys or add these two back to the messages file

Reply:
Put back in 753414651f6664b310706888504b0ce1965532bb